helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestWagedRebalancer.java

```diff
@@ -379,13 +379,13 @@ public void testRebalanceOnChanges() throws IOException, HelixRebalanceException
 
     // 2. rebalance with one ideal state changed only
```

Comment:
Is this "2." still valid? Now we seem to have a ResourceConfig with change.

Reply:
Will update.